app/helpers/navigation_items.rb

```diff
@@ -31,7 +31,12 @@ def for_support_primary_nav(current_support_user, current_controller)
     end
 
     def for_support_account_nav(current_support_user)
-      if current_support_user
+      if (impersonated_user = current_support_user&.impersonated_provider_user)
+        [
+          NavigationItem.new("🎭 #{current_support_user.email_address}", support_interface_provider_user_path(impersonated_user), false),
```

Comment:
Emoji can cause accessibility issues, as their (sometime non-intuitive) Unicode name is announced. We should output something like this:

`<span aria-hidden="true">🎭</span><span class="govuk-visually-hidden">Impersonating:</span>`

Reply:
Noted, thanks. I've added such span elements.